development-environment.md

```diff
@@ -2,7 +2,7 @@
 In order to reduce the amount of effort of setting up development environments and fixing versioning between machines, a dev-container is provided which works with the Remote-Container functionality in Visual Studio code.
 
 ## Prerequisites
-* [Docker](https://www.docker.com/products/docker-desktop)
+* [Colima](https://github.com/abiosoft/colima) This is an alternative to [Docker Desktop](https://www.docker.com/products/docker-desktop) which requires license. Use Docker desktop only if your client project demands it and provide license.
```

Comment:
We should keep this free of the word "client". We don't actually care what people use, so long as they get some sort of containerisation that is proper to their personal situations.

* [Docker Desktop](https://www.docker.com/products/docker-desktop) or [Colima](https://github.com/abiosoft/colima) (as an alternative if your use case does not meet the [Docker Subscription Service Agreement](https://www.docker.com/legal/docker-subscription-service-agreement/)